src/System.Management.Automation/engine/serialization.cs

```diff
@@ -1109,46 +1109,39 @@ private bool HandleSecureString(object source, string streamName, string propert
                     // the principle used in serialization is that serialization
```

Comment:
Please update comment to explain why we are throwing from serializer in this case when normally that is not allowed.

Reply:
Will update